app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var run = require('./cli/lib/gateway')();
+const os = require('os');
+
+const options = {};
+
+options.env = process.env.EDGEMICRO_ENV;
+options.key = process.env.EDGEMICRO_KEY;
+options.secret = process.env.EDGEMICRO_SECRET;
+options.org = process.env.EDGEMICRO_ORG;
+options.configDir = process.env.EDGEMICRO_CONFIG_DIR;
+options.processes = os.cpus().length;
```

Comment:
This should probably accept the `process.env.EDGEMICRO_PROCESSES` environment variable, and default to number of cpus.